<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashWidget</class>
 <widget class="QWidget" name="SplashWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QVideoWidget" name="widgetVideo" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelProductName">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: bold 20pt &quot;微软雅黑&quot;;</string>
            </property>
            <property name="text">
             <string>Revealer Control Center</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelProductVersion">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
            </property>
            <property name="text">
             <string>V2.0.0.1234.20210323</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelCompanyName">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>FuHuang AgileDevice</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButtonSkip">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>Skip(5s)</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>QVideoWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
